app/resources/css-default.template

```diff
@@ -122,6 +116,7 @@ pre {  /* code samples */
 
 .button-blue input {
   background: #4285f4;
+  margin-top: 10px;
```

Comment:
Why did you add margin here? Maybe you should also add it to .button-red as well? Why not adding margin to the bottom as well?